src/annotator/highlighter.js

```diff
@@ -326,9 +326,13 @@ export function removeHighlights(highlights) {
  * @param {boolean} focused
  */
 export function setHighlightsFocused(highlights, focused) {
-  highlights.forEach(h =>
-    h.classList.toggle('hypothesis-highlight-focused', focused)
-  );
+  highlights.forEach(h => {
+    if (h.svgHighlight) {
```

Comment:
If this only applies to PDFs, then it might helpful to point that out in comment here, and likewise for the else case for HTML.